test/Microsoft.ML.Tests/OnnxTests.cs

```diff
@@ -329,7 +329,7 @@ public void KmeansTest()
             fileText = Regex.Replace(fileText, "\"producerVersion\": \"([^\"]+)\"", "\"producerVersion\": \"##VERSION##\"");
             File.WriteAllText(onnxAsJsonPath, fileText);
 
-            CheckEquality(subDir, "Kmeans.json");
+            CheckEquality(subDir, "Kmeans.json", digitsOfPrecision: 2);
```

Comment:
2 digits of precision is getting pretty low, isn't it?

Reply:
yes so should i revert it ?